### DISCUSSION TITLE
Would there be a way to disable ghost previews?

### DISCUSSION THREAD

**Discussion by penolakushari** [wiremod/advdupe2#350] [Suggestions]
Perhaps a bit of an unusual question, but I happen to do some renders with "soft lamps" with some contraptions that I save with advdupe2. 

From what I could tell, seems like when props are being ghosted by the tool and then pasted, engine still seems to think that they are transparent and makes them not cast shadows from projected lights (Lamps, flashlights), which also seems to affect other props with the same model that I spawn in afterwards through spawnmenu. Restarting the game fixes all the problems until I spawn the contraption again with ghosting on.

So, perhaps having a checkbox to disable ghosting when selecting a dupe would help, considering that on older version of advdupe2 that I had setting ghost percentage to 0 makes it so all props can cast shadows normally, although it still ghosted at least 1 prop from the dupe.

**Comment by thegrb93** [comment]
I'll see about adding back the ghost percent slider

**Reply by thegrb93** [reply]
https://github.com/wiremod/advdupe2/commit/162b052082857f4e5f6d7ba1cefa8f4ce4f3e0c4

**Comment by thegrb93** [comment]
Ok, ghost limit is added back

**Reply by penolakushari** [reply]
Many thanks!
